helm/csi-isilon/templates/controller.yaml

```diff
@@ -314,7 +314,7 @@ spec:
         {{end}}
         {{- end }}
         - name: provisioner{{ $csiSidecarSuffix }}
-          image: {{ required "Must provide the CSI provisioner container image." .Values.images.provisioner }}
+          image: {{ required "Must provide the CSI provisioner container image." ( include "csi-isilon.provisionerImage" . ) }}
```

Comment:
adjust spacing before "(" and after ")" , should be consistent across all sidecars